pdr_backend/data_eng/test/resources.py

```diff
@@ -30,9 +32,21 @@ def _data_pp_ss_1feed(tmpdir, feed, st_timestr=None, fin_timestr=None):
     parquet_dir = str(tmpdir)
     pp = _data_pp([feed])
     ss = _data_ss(parquet_dir, [feed], st_timestr, fin_timestr)
-    pq_data_factory = OhlcvDataFactory(pp, ss)
+    ohlcv_data_factory = OhlcvDataFactory(pp, ss)
```

Comment:
I'd like to avoid this pattern if possible... 
i.e. if we're contributing towards the same branch, fork the branch and push a PR with that branch as base.

I specifically did this here rather than in yaml-cli2, such that you can review my changes rather than being unaware that I did that.